src/sync/planner.rs

```diff
@@ -579,6 +579,11 @@ impl SeafowlSyncPlanner {
         )?;
         let pruning_predicate =
             PruningPredicate::try_new(prune_expr, full_schema.clone())?;
+        trace!(
+            "Constructed pruning predicate: {:?}, schema {}",
+            pruning_predicate,
```

Comment:
Putting this as trace because it's pretty verbose and contains actual data from the tables (min/max values)